### ISSUE TITLE
Installing Brigadier.NET seems to pull the entire standard library

### ISSUE THREAD

**Issue by RedstoneParadox** [AtomicBlom/Brigadier.NET#2]
Attempted this with both a Mono-based and a .NET based project - and then attempting to download and install from my computer directly - and it seems to me that every package in the entire standard library is listed as a dependency and gets pulled. I have gone into my packages folder and verified that these packages are indeed getting placed there.

Edit: I'm not sure it's because I'm doing something wrong, but I did try several other packages to make sure I was doing things right and none of them pulled the stdlib.

Edit 2: Forgot to mention that the Mono-based project is on Mono 5.18.

**Comment by AtomicBlom** [comment]
It's very possible I've stuffed something up. It's my first NuGet package so I'm still trying to figure out how it all works.

I used the NuGet package stuff that's built into the new .csproj format and maybe that was a bad idea. I'll do some research and try get it fixed.

**Comment by RedstoneParadox** [comment]
Found this in the .csproj

```xml
<ItemGroup>
    <PackageReference Include="System.Memory" Version="4.5.1" />
</ItemGroup>
```

And all the dependencies I'm seeing are listed as dependencies or dependencies of the project's dependencies, etc. so that may be what's dragging in all the dependencies. Will Brigadier.NET not compile without this?

Edit: So I looked into it and it seems to have to do with the .NET standard before v2.0: https://stackoverflow.com/questions/47365136/why-does-my-net-standard-nuget-package-trigger-so-many-dependencies
